apps/mobile/src/components/Markdown.tsx

```diff
@@ -79,6 +79,12 @@ export function Markdown({
           {children}
         </Text>
       );
+
```

Comment:
what does this do?

Reply:
Its fixed the markdown with `multiline` props. Removed the hardcoded props that I used